desc/objectives/objective_funs.py

```diff
@@ -816,10 +848,6 @@ def __init__(
 
     def _set_derivatives(self):
```

Comment:
Same comment as above, I would prefer having this in init function.

Reply:
see comment above